Prox/Prox/PlaceCarousel/PlaceCarouselViewController.swift

```diff
@@ -207,7 +212,7 @@ class PlaceCarouselViewController: UIViewController {
         // this ensures consistency between the place details and the carousel underneath
         // and makes sure we don't end up providing weird data to users while they are scrolling
         // through places details
-        updatePlaces(forLocation: location)
+        updatePlaces(for: location)
```

Comment:
nit: Not sure I like this arg name change. What if we'd decided to call `location` `fred`? Suddenly the meaning derived from the parameter name is lost and the function call is less explicit.